<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMenu</class>
 <widget class="QWidget" name="PageMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择菜单</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="text">
         <string>主界面</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="text">
         <string>人脸录入</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="text">
         <string>历史记录</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="text">
         <string>数据分析</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>446</width>
          <height>62</height>
         </size>
        </property>
        <property name="text">
         <string>帮助文档</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
